code/src/components/Header.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const Header = ({ api, onApiChange }) => {
+  return (
+    <header>
```

Comment:
This is great. For sure something I should add onto when I do my code next time.